packages/host/tests/acceptance/code-mode-test.ts

```diff
@@ -836,6 +836,36 @@ module('Acceptance | code mode tests', function (hooks) {
       .exists();
 
     assert.dom(`[data-test-card-schema="Base"]`).exists();
+
+    // Check that realm icons in the schema editor are correct (card and its fields)
+
+    let realm1IconUrl = 'https://i.postimg.cc/L8yXRvws/icon.png';
+    let realm2IconUrl = 'https://i.postimg.cc/d0B9qMvy/icon.png';
```

Comment:
Could you get these by importing from `packages/drafts-realm/.realm.json` etc? Not a big deal as these will likely not change (often?) but sometimes nice to have strings not copied across an application multiple times.

Reply:
Ah good idea, will try that.